@shared/api/helpers.ts

```diff
@@ -39,13 +40,14 @@ export const sendMessageToContentScript = (msg: {}): Promise<Response> => {
   });
 };
 
-export const sendMessageToBackground = (msg: {}): Promise<Response> => {
+export const sendMessageToBackground = async (msg: {}): Promise<Response> => {
```

Comment:
`webextension-polyfill` uses a promise based API rather than the callback based API we were using, so all these methods had to be mildly refactored